src/index.html

```diff
@@ -1,13 +1,61 @@
+<!--#region head -->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
+  <!--#endregion -->
+
   <body>
-    <h1>Product cards</h1>
+    <!--#region card head -->
+    <div class="product-card" data-qa="card">
+      <img
+      src="./images/imac.jpeg"
+      alt="apple A1419 imac-27 Retina 5K monoblock"
+      class="product-card__image"
+      >
+      <h1 class="product-card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <h2 class="product-card__product-code">Product code: 195434</h2>
+      <!--#endregion -->
+      <!--#region card reviews-->
+      <div class="reviews">
+        <div class="reviews__stars">
+          <div class="reviews__star star"></div>
+          <div class="reviews__star star"></div>
+          <div class="reviews__star star"></div>
+          <div class="reviews__star star"></div>
+          <div class="reviews__star star"></div>
+        </div>
```

Comment:
Just use the "stars" block from the previous task, you don't need any new "review" block for this task.

Checklist:
5. [TASK] - Make sure to reuse part of [Stars task](https://github.com/mate-academy/layout_stars) here, and keep it operational in case you change block modifier.
6. [BEM] - Don't make BEM structure too complicated. You don't need to create separate blocks for anything other than card itself and stars.

Reply:
Looks like you forgot to process the comment above.